lib/cc/engine/analyzers/analyzer_base.rb

```diff
@@ -9,7 +9,11 @@ def initialize(engine_config:, directory:)
 
         def run
           files.map do |file|
-            process_file(file)
+            begin
+              process_file(file)
+            rescue => e
+              $stderr.puts e
```

Comment:
The previous comment is what your output looks like.

This is what it currently looks like:

```
../paperclip/lib/paperclip/attachment.rb:7 :: parse error on value 1 (tINTEGER)
```